app/celery/scheduled_tasks.py

```diff
@@ -228,6 +228,16 @@ def recover_expired_notifications():
     sms_queue.expire_inflights()
     email_queue.expire_inflights()
```

Comment:
probably should have these two in an "else" after the FF_PRIORITY_LANES "else" below, as you did for the init_app()